src/hotspot/share/gc/g1/g1FullGCMarker.hpp

```diff
@@ -63,6 +63,9 @@ class G1FullGCMarker : public CHeapObj<mtGC> {
   G1FollowStackClosure _stack_closure;
   CLDToOopClosure      _cld_closure;
 
+
+  G1RegionMarkStatsCache _mark_region_cache;
```

Comment:
Please add the include to this file: 
#include "gc/g1/g1RegionMarkStatsCache.hpp"

Reply:
Thanks Stefan, Just modified